solutions/devsprint-firas-safa-1/GitHubApp/Screens/List/ListViewModel.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+struct ListViewModel {
+    private var listItems: [String]
+    
+    init(listItems: [String] = []) {
+        self.listItems = listItems
+    }
+    
+    var rowCount: Int {
+        listItems.count
+    }
```

Comment:
```suggestion
struct ListViewModel {
    private var listItems: [String]
     
    var rowCount: Int {
        listItems.count
    }
    
    init(listItems: [String] = []) {
        self.listItems = listItems
    }
```

as a general rule we do:

- stored properties
- computed properties
- init
- functions

this is a nice order to have to order the files